recipes/holodeck-gw/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "holodeck-gw" %}
+{% set version = "1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/holodeck-gw-{{ version }}.tar.gz
+  sha256: 6a20222953caaf22a98488a0de60a9cc74f97cc4ef912450ece4c5d02ff6a135
+
+build:
+  skip: true  # [py<38 or win]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - setuptools >=40.6.0
+    - wheel
+    - cython <3.0.0
+    - numpy <2.0.0
+    - scipy
+    - numba
+    - pip
+    - mpi4py
+  run:
+    - python
+    - kalepy
+    - astropy
+    - cosmopy
+    - h5py
+    - matplotlib-base
+    - scipy
+    - pydoe
+    - psutil
+    - ipywidgets
+    - tqdm
+    - cython <3.0.0
```

Comment:
It is unlike that you need cython as a run requirement. Can you check this upstream?

Reply:
The upstream code requires `cython` as a runtime dependency https://github.com/nanograv/holodeck/blob/main/requirements.txt

Reply:
It lists but is it used? Does this package generate code?

Reply:
I could not find a place where cython was used beyond the building step. Maybe I'm missing something.

Reply:
The upstream code hasn't finished migrating docs and such to `pyproject.toml`. The current `setup.py` based installation uses `requirements.txt` for both build and runtime requirements. The PyPI package therefore requires `cython` as a runtime dependency. If `cython` isn't listed as a runtime dependency, then the build will fail (I've confirmed with local builds).

In the future, the code will move to pure `pyproject.toml` and `cython` can be removed from runtime dependencies as you've pointed out.

Reply:
You mean the `pip check` test will fail, not the build, right? If so it is preferred to remove cython and comment out the `pip check` test to avoid polluting user env with build tools.

PS: one can have cython as a build tool only without a pyproct.toml.

Reply:
I've removed the cython run time dependency and the pip check as requested